ffmpeg.c

```diff
@@ -789,6 +853,15 @@ static int ffmpeg_set_picture(struct ffmpeg *ffmpeg){
     ffmpeg->picture->width  = ffmpeg->ctx_codec->width;
     ffmpeg->picture->height = ffmpeg->ctx_codec->height;
 
+    // h264_v4l2m2m encoder expects video buffer to be allocated
+    if (strcmp(ffmpeg->codec->name, "h264_v4l2m2m") == 0) {
+        if (alloc_video_buffer(ffmpeg->picture, 32)) {
+            MOTION_LOG(ERR, TYPE_ENCODER, NO_ERRNO, _("could not alloc buffers"));
```

Comment:
There are multiple possible error returns in the `alloc_video_buffer`  function.  Is it going to be helpful to report the error or possibly report which one triggered the fail inside of the `alloc_video_buffer` function?  (This is a question rather than a request.  I do not know the likelihood of any of them failing).

Reply:
Would it help if I add something like `av_strerror(retcd, errstr, sizeof(errstr));` just before the MOTION_LOG line?